### ISSUE TITLE
erro

### ISSUE THREAD

**Issue by DarshJoshi6075** [swapnilbamble1438/EcommerceApp#1]
brother i am getting error after following every step like you said error says "the import org.apache.tomcat cannot be resolved" how to solve please help

**Comment by swapnilbamble1438** [comment]
Try redownloading tomcat 8.5 server.
and send me screenshot of the error

**Comment by DarshJoshi6075** [comment]
Broo thanks it worked but i couldn't find application. Properties so can
you help me to find and give me connection to the database and one question
this project have conncetion with database or not ?

**Comment by swapnilbamble1438** [comment]
it is J2EE application it does not have application.properties file and you
dont need to create any database or table. Beacuae I have used Sqlite
database. so the database file(madabase.db) is in the app folder only. but
you have to so some path changes. Go to com.conn pacakge you will find
DBConnect.java file there in line no. 17 you have to change the path
according to your mydatabase.db file location.

**Comment by DarshJoshi6075** [comment]
Yes bro got it thank you for replying and sorry for disturbing but i want
to change the home page containing online shopping application text  how to
do it or it is an img.

**Comment by DarshJoshi6075** [comment]
Bro is there any functionality to change the status of order like
processing to delivered or anything ?

**Comment by swapnilbamble1438** [comment]
sorry bro currently I am working on some other project. If I'll get time, I
will try to do that change and I will inform you about it.

**Comment by DarshJoshi6075** [comment]
Its okay brother no issues.

**Comment by presays07** [comment]
Can I get the db file for this project? The one in the App project has glitchy unreadable characters similar to when we open a pdf in a word file.
